common/components/common/projects/ContactProjectButton.jsx

```diff
@@ -185,10 +185,8 @@ class ContactProjectButton extends React.PureComponent<Props, State> {
     if (this.state) {
       if (CurrentUser.isLoggedIn()) {
         return (
-          <div>
-            {this.displayEditProjectButton()}
-            {this.displayContactProjectButton()}
-          </div>
+          this.displayEditProjectButton(), 
+          this.displayContactProjectButton()
```

Comment:
```suggestion
          <React.Fragment>
            {this.displayEditProjectButton()} 
            {this.displayContactProjectButton()}
          </React.Fragment>
```

The render function needs to return a react element.  If you need to return multiple elements without an enclosing <div>, you can use React.Fragment.